tests/src/main/java/org/hibernate/beanvalidation/tck/tests/validation/ValidateValueTest.java

```diff
@@ -178,7 +179,7 @@ public void testValidIsNotHonoredValidateValue() {
 		customer.addOrder( order );
```

Comment:
Given this is testing `validateValue()`, the `customer` instance could be removed altogether.

Reply:
Sure, I can make that change if you like.